src/renderer/components/top-nav/top-nav.vue

```diff
@@ -44,7 +44,14 @@
         :title="newWindowText"
         @click="createNewWindow"
       />
-      <div class="logo">
+      <div
+        class="logo"
+        role="link"
+        tabindex="0"
+        :title="$t('Subscriptions.Subscriptions')"
+        @click="navigate('subscriptions')"
+        @keypress="navigate('subscriptions')"
```

Comment:
'keypress' is deprecated and triggers on many keys. Prefer the usage of 'keydown'. Because it is acting as a link, it should trigger only on the Enter key. One easy way to implement this behavior would be @keydown.enter.prevent="...".